scanner/src/funTest/kotlin/scanners/ScannerIntegrationFunTest.kt

```diff
@@ -48,28 +48,31 @@ import org.ossreviewtoolkit.utils.test.getAssetFile
 import org.ossreviewtoolkit.utils.test.matchExpectedResult
 import org.ossreviewtoolkit.utils.test.patchActualResult
 
-class ScannerIntegrationFunTest : StringSpec({
-    "scan() returns the expected ORT result for a given analyzer result".config(invocations = 3) {
-        val analyzerResultFile = getAssetFile("analyzer-result.yml")
-        val expectedResultFile = getAssetFile("dummy-expected-output-for-analyzer-result.yml")
+class ScannerIntegrationFunTest : WordSpec({
+    "scan()" should {
```

Comment:
Is this a preparation for testing also other functions here? If so, does it make sense to mention that in the commit message?